renderdoc/driver/gl/gl_dispatch_table.h

```diff
@@ -986,7 +986,7 @@ void SetDriverForHooks(WrappedOpenGL *driver);
 // best we can do.
 // On apple we suppress hooks while entering any CGL function so we don't record internal work that
 // can mess up the replay
-#if ENABLED(RDOC_WIN32) || ENABLED(RDOC_APPLE)
+#if ENABLED(RDOC_WIN32) || ENABLED(RDOC_APPLE) || defined(RDOC_SWITCH)
```

Comment:
One thing you need to be careful of with these is that `defined(RDOC_SWITCH)` is always true, because we always define it either to `OPTION_ON` or `OPTION_OFF`. You need to use `ENABLED()` or `DISABLED()` as with the other checks.

The reason is mostly to avoid the ambiguity with different conventions where if a macro is defined as 0/1 but then checked with `defined()` it will always be counted as true. The `ENABLED()` and `DISABLED()` checks will fail to compile unless the parameter is defined to either `OPTION_ON` or `OPTION_OFF`, and fail to compile if it's undefined completely.

Reply:
Sorry, I did mean to change this and overlooked it.